src/ProjectOrigin.WalletSystem.Server/Services/REST/v1/ClaimsController.cs

```diff
@@ -117,13 +119,48 @@ public class ClaimsController : ControllerBase
             Quantity = request.Quantity,
         };
 
+        await unitOfWork.RequestStatusRepository.InsertRequestStatus(new Models.RequestStatus
+        {
+            RequestId = command.ClaimId,
+            Status = RequestStatusState.Pending
+        });
+
         await bus.Publish(command);
 
+        unitOfWork.Commit();
+
         return Accepted(new ClaimResponse()
         {
```

Comment:
Can you add a loication header with the status endpoint?

Reply:
done